packages/cinterop/src/jvmMain/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.4.1)
```

Comment:
I left this on purpose to see how this will end up looking once I added Win/Linux either have one CMake with multiple targets or do a modular one or don't use Cmake at all (just a docker call to build the remaining platform). 

Note that the JNI header file and linker flags are slightly different between JVM and Android